src/main/blackbox/blackbox.c

```diff
@@ -855,6 +869,13 @@ static void writeInterframe(void)
             blackboxWriteSignedVB(blackboxCurrent->servo[5] - blackboxLast->servo[5]);
         }
     }
+
+    if (isFieldEnabled(FIELD_SELECT(SERVO))) {
+        for (int x=0; x<MAX_SUPPORTED_SERVOS; ++x) {
```

Comment:
- Would suggest adding `getServoCount()`
- Does this log servo[5] twice when TRICOPTER mixer is used?
```suggestion
        for (int x = 0; x < MAX_SUPPORTED_SERVOS; x++) {
```